changes/2899.fix.md

```diff
@@ -0,0 +1 @@
+Add the missing `id` field to the `ContainerRegistryNode.from_row()`.
```

Comment:
```
Assign valid value to `id` field in `ContainerRegistryNode` GQL schema query resolver
```
This would be better since we assign value here rather than add the field in the schema